src/main/java/tk/shanebee/hg/events/PlayerJoinGameEvent.java

```diff
@@ -20,6 +23,7 @@ public PlayerJoinGameEvent(Game game, Player player) {
 		this.game = game;
 		this.player = player;
 		this.isCancelled = false;
+
```

Comment:
I'm not sure why this blank line was added

In general this class doesn't appear to actually be changed and/or need a change